var/spack/repos/builtin/packages/openturns/package.py

```diff
@@ -38,6 +40,7 @@ class Openturns(CMakePackage):
     depends_on("intel-tbb", type=("build", "run"))
     depends_on("py-cloudpickle", type=("build", "run"))
     depends_on("py-urllib3", type=("build", "run"))
+    depends_on("libxml2", type=("build", "run"))
```

Comment:
@mschouler is the libxml2 dependency new as of a specific version, or was this something we'd previously been missing and is required for all versions of the package?

Reply:
It is definitely not a new dependency but I don't think it is a "must-have" dependency either (see [CMake options](https://github.com/openturns/openturns/blob/master/CMakeLists.txt#L18)). If you think it is more appropriate I can make it a variant instead.

Reply:
Oh gotcha, do you think most people would want to enable xml support in openturns? If not I'd probably recommend turning it into a variant.

Reply:
That's tough to say but since in doubt I'll make it a variant then.